apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketList/components/BottomSection/BottomSectionCont.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useMarketData } from "@ledgerhq/live-common/market/MarketDataProvider";
+import { MarketListRequestParams } from "@ledgerhq/live-common/market/types";
+import {
+  marketFilterByStarredAccountsSelector,
+  starredMarketCoinsSelector,
+} from "~/reducers/settings";
+import { track } from "~/analytics";
+import { setMarketFilterByStarredAccounts, setMarketRequestParams } from "~/actions/settings";
+import { getAnalyticsProperties } from "LLM/features/Market/utils";
+import BottomSection from "./BottomSection";
+
+function BottomSectionCont() {
+  const dispatch = useDispatch();
+  const { requestParams, counterCurrency, refresh } = useMarketData();
+  const { range, orderBy, order, top100 } = requestParams;
+  const starredMarketCoins: string[] = useSelector(starredMarketCoinsSelector);
+  const filterByStarredAccount: boolean = useSelector(marketFilterByStarredAccountsSelector);
+  const firstMount = useRef(true); // To known if this is the first mount of the page
+
+  useEffect(() => {
+    if (firstMount.current) {
+      // We don't want to refresh the market data directly on mount, the data is already refreshed with wanted parameters from MarketDataProviderWrapper
+      firstMount.current = false;
```

Comment:
it's an antipattern of react effect hooks, there may be better ways to treat this in the future and bailout from the 'first' case